resources/packs/aws/aws_ec2.go

```diff
@@ -301,6 +301,40 @@ func (s *mqlAwsEc2) getSecurityGroups(provider *aws_provider.Provider) []*jobpoo
 
 						mqlIpPermissions = append(mqlIpPermissions, mqlSecurityGroupIpPermission)
 					}
+					mqlIpPermissionsEgress := []interface{}{}
+
+					for p := range group.IpPermissionsEgress {
+						permission := group.IpPermissionsEgress[p]
+
+						ipRanges := []interface{}{}
+						for r := range permission.IpRanges {
+							iprange := permission.IpRanges[r]
+							if iprange.CidrIp != nil {
+								ipRanges = append(ipRanges, *iprange.CidrIp)
+							}
+						}
+
+						ipv6Ranges := []interface{}{}
+						for r := range permission.Ipv6Ranges {
+							iprange := permission.Ipv6Ranges[r]
+							if iprange.CidrIpv6 != nil {
+								ipRanges = append(ipRanges, *iprange.CidrIpv6)
+							}
+						}
+						mqlSecurityGroupIpPermission, err := s.MotorRuntime.CreateResource("aws.ec2.securitygroup.ippermission",
+							"id", core.ToString(group.GroupId)+"-"+strconv.Itoa(p),
```

Comment:
```suggestion
							"id", core.ToString(group.GroupId)+"-"+strconv.Itoa(p)+"-egress",
```

I had a similar problem with the networkAcls: https://github.com/mondoohq/cnquery/pull/1508

Reply:
oh thank you!